playbooks/common/openshift-cluster/update_repos_and_packages.yml

```diff
@@ -8,6 +8,5 @@
           ansible_distribution == "RedHat" and
           lookup('oo_option', 'rhel_skip_subscription') | default(rhsub_skip, True) |
             default('no', True) | lower in ['no', 'false']
```

Comment:
@detiber I have rebased this PR and the deployment_type check is now checking the 3 values as expected.